controllers/Passport.js

```diff
@@ -0,0 +1,88 @@
+"use strict";
+
+const passport = require("passport");
+
+class Passport {
+	constructor(app, db) {
+		this.passport = passport;
+		this.createPassportSession(app, db);
+	}
+
+	createPassportSession(app, db) {
+		app.use(passport.initialize(undefined));
+		app.use(passport.session(undefined));
+
+		// use `id` for serializing and deserializing users in database
+		passport.serializeUser(function (user, done) {
+			console.log("Serializing " + user.id);
```

Comment:
```suggestion
```
These won't be accessible, we better create some form of logger that writes to the disc.

Reply:
Done together with the next suggestion as a single commit.